draft-ietf-tls-tls13.md

```diff
@@ -1008,7 +1020,16 @@ of {{RFC5116}}. The key is either the client_write_key or the server_write_key.
            uint16 length;
            opaque nonce_explicit[SecurityParameters.record_iv_length];
            aead-ciphered struct {
-              opaque content[TLSPlaintext.length];
+               select(type) {
+                   case alert:
+                   case application_data:
+                       opaque content[TLSPlaintext.length];
+                   case handshake:
+                   case application_data_padded:
+                       opaque padding<0..2^16-1>;
+                       opaque content[TLSPlaintext.length -
+                                      (2 + length(padding))];
```

Comment:
If I'm not misunderstanding things, this should be TLSCiphertext.length - 2 - length(padding), which is equal to TLSPlaintext.length (not plain, as it currently is on line 1030). I think it would be more clear to just have the padding element under a conditional and have only the one content[TLSPlaintext.length], with a note on how TLSPlaintext.length is determined.